Formula/cfengine.rb

```diff
@@ -20,12 +20,11 @@ class Cfengine < Formula
   depends_on "openssl"
 
   resource "masterfiles" do
-    url "https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-masterfiles-3.8.0.tar.gz"
-    sha256 "6956ba4a359e8fe03b627b3fb16b382fed6e33cdfc303db08fb9790895c2a98e"
+    url "https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-masterfiles-3.9.0.tar.gz"
+    sha256 "63dec2f8649f5f2788cd463dccf47f8dbe941522acfcf3093517f983bbfa0606"
   end
 
   def install
-    system "autoreconf", "-Wno-portability", "-fvi", "-I", "m4" # see autogen.sh
```

Comment:
If we no longer need this step can we get rid of the autotool `:build` dependencies?